### PR TITLE
Switching icon, adding website

### PR DESCRIPTION
**Pull request by Duncid**
## Description
Small polishing pass on displaying assistant usage in Folders and Websites

### CODE REVIEW

File: front/pages/w/[wId]/builder/data-sources/public-urls.tsx [hunk 1]
```diff
@@ -213,12 +214,16 @@ export default function DataSourcesView({
...
-                  <span className="px-1 text-element-500">|</span>
-                  <span>{dataSourcesUsage[ds.id] ?? 0}</span>
-                  <RobotStrokeIcon />
-                </div>
+                <>
+                  <>
+                    Added by: {ds.editedByUser?.fullName}
+                    <span className="mx-2 h-3 w-0.5 bg-element-500" />
+                    {dataSourcesUsage[ds.id] ?? 0}
+                  </>
```
Comment (JulesBelveze) [inline front/pages/w/[wId]/builder/data-sources/public-urls.tsx:222]: This is duplicated 👍🏼

**Comment by Duncid** [comment]
@JulesBelveze should be good for review!

**Review by JulesBelveze — COMMENTED** [review]
That's a screenshot from my monitor, looks a bit broken when urls are long, no?

Screenshot 2024-07-09 at 15 43 17

**Comment by Duncid** [comment]
@JulesBelveze Already the case today indeed:
image

IMO ship this, fix the component after!

**Comment by JulesBelveze** [comment]
Fine with me 👍🏼

File: front/pages/w/[wId]/builder/data-sources/public-urls.tsx [hunk 2]
```diff
@@ -213,12 +214,12 @@ export default function DataSourcesView({
...
-                <div className="flex items-center gap-1 pl-2 pt-0.5 text-xs text-element-700">
-                  <span>Added by: {ds.editedByUser?.fullName}</span>
-                  <span className="px-1 text-element-500">|</span>
-                  <span>{dataSourcesUsage[ds.id] ?? 0}</span>
-                  <RobotStrokeIcon />
-                </div>
+                <>
+                  Added by: {ds.editedByUser?.fullName}
+                  <span className="mx-2 h-3 w-0.5 bg-element-500" />
+                  Used by: {dataSourcesUsage[ds.id] ?? 0}
```
Comment (JulesBelveze) [inline front/pages/w/[wId]/builder/data-sources/public-urls.tsx:220]: `nit`: I would probably wrap text in `<span>` tags even though text doesn't require styling

File: front/pages/w/[wId]/builder/data-sources/static.tsx [hunk 3]
```diff
@@ -164,12 +165,12 @@ export default function DataSourcesView({
...
-                <div className="flex items-center gap-1 pl-2 pt-0.5 text-xs text-element-700">
-                  <span>Added by: {ds.editedByUser?.fullName}</span>
-                  <span className="px-1 text-element-500">|</span>
-                  <span>{dataSourcesUsage[ds.id] ?? 0}</span>
-                  <RobotStrokeIcon />
-                </div>
+                <>
+                  Added by: {ds.editedByUser?.fullName}
+                  <span className="mx-2 h-3 w-0.5 bg-element-500" />
+                  Used by: {dataSourcesUsage[ds.id] ?? 0}
```
Comment (JulesBelveze) [inline front/pages/w/[wId]/builder/data-sources/static.tsx:171]: same here

**Review by JulesBelveze — APPROVED** [review]
LGTM ! 

Left a couple of nits

**Comment by JulesBelveze** [comment]
FYI @Duncid there's already a card to fix the `ContextItem` component here: https://github.com/dust-tt/tasks/issues/1035

**Pull request merged**